diff --git a/backbone/resnet_irse.py b/backbone/resnet_irse.py
index eb7f7b3..93aa61a 100644
--- a/backbone/resnet_irse.py
+++ b/backbone/resnet_irse.py
@@ -138,14 +138,13 @@ class Backbone(Module):
         self._initialize_weights()
 
     def forward(self, x):
-        with torch.cuda.amp.autocast(cfg["ENABLE_AMP"]):
-            x = self.input_layer(x)
-            x = self.body(x)
-            # x = self.output_layer(x)
-            x = self.bn2(x)
-            x = torch.flatten(x, 1)
-            x = self.dropout(x)
-        x = self.fc(x.float() if cfg["ENABLE_AMP"] else x)
+        x = self.input_layer(x)
+        x = self.body(x)
+        # x = self.output_layer(x)
+        x = self.bn2(x)
+        x = torch.flatten(x, 1)
+        x = self.dropout(x)
+        x = self.fc(x)
         x = self.features(x)
 
         return x
